ansible/roles/common/tasks/main.yml

```diff
@@ -70,7 +70,7 @@
   changed_when: false
 
 - name: ensure service user {{user}} exists
-  user: name={{user}} system=yes createhome=no home=/nonexistent
+  user: name={{user}} system=yes createhome=yes home=/home/{{user}}
```

Comment:
This user does actually have to exist, as `{{user}}` is used later in the `warcproxy` playbook. The only reason this didn't error before is because `/home/vagrant` is created by default by Vagrant itself. (I was running inside Vagrant though, so I hit this).